services/core-web/src/components/common/RenderRadioButtons.tsx

```diff
@@ -46,8 +47,17 @@ const RenderRadioButtons: FC<RenderRadioButtonsProps> = (props) => {
         value={input.value}
         onChange={handleRadioChange}
         id={id as string}
-        options={options}
-      />
+      >
```

Comment:
the common version has isVertical instead of vertical for a parameter. Just something to keep in mind